examples/amp-story/attachment.html

```diff
@@ -174,8 +181,14 @@
           cta-text="Subscribe">
           <div class="attachment-content">
             <p>Form within a small attachment:</p>
-            <form>
+            <form
+              method="POST"
+              action-xhr="https://www.example.com"
+              target="_top"
```

Comment:
Do we have to enforce this is always `_top` and never `_blank`? What happens if you specify `_blank`?

Reply:
`_blank` supposedly opens the submission response in a new window or tab, which may not be a flow we want to allow.

I haven't been able to test amp-form with a valid endpoint (because my local server uses HTTP and `amp-form` requires and HTTPS action), so I can't yet confirm the behavior of `_blank` within a story. I'll observe the difference in behavior when I create my Firebase demo, and then we can enforce `_top` if necessary